AmplifyPlugins/Auth/Podfile.lock

```diff
@@ -26,7 +26,7 @@ PODS:
   - CwlCatchException (1.0.2)
   - CwlPreconditionTesting (1.1.1):
     - CwlCatchException
-  - SwiftFormat/CLI (0.44.13)
+  - SwiftFormat/CLI (0.44.14)
```

Comment:
Don't we also need to declare a dependency on AWSMobileClient `~> 2.13.5`?

Reply:
Updated